package.json

```diff
@@ -21,19 +21,25 @@
     "@fortawesome/free-solid-svg-icons": "^6.5.1",
     "@fortawesome/react-fontawesome": "^0.2.0",
     "@radix-ui/react-dropdown-menu": "^2.0.6",
+    "@radix-ui/react-slot": "^1.0.2",
     "axios": "^1.6.7",
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.0",
     "cryptr": "^6.3.0",
     "date-fns": "^3.3.1",
+    "embla-carousel-react": "^8.0.0",
     "iso-639-3": "^3.0.1",
     "jwt-decode": "^4.0.0",
+    "lucide-react": "^0.343.0",
     "next": "14.1.0",
     "next-auth": "^4.24.5",
     "react": "^18",
     "react-dom": "^18",
+    "react-icons": "^5.0.1",
```

Comment:
Can you check where does these `react-icons` package is being used?

Reply:
Hey, let me double check this imports here because these are from a push I mistakenly did so we could showcase Homepage today in the morning. Had completely forgot that the pr I opened on Friday was still not merged.